qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -743,17 +743,18 @@ wait_for_journal_replay_complete()
 {
     local local_cluster=$1
     local cluster=$2
-    local pool=$3
-    local image=$4
+    local local_pool=$3
+    local remote_pool=$4
```

Comment:
I think this failure in `TEST: start first daemon instance and test replay` in `rbd_mirror_ha.sh` workunit is related.  The call stack is
```
test_replay test1
  wait_for_replay_complete
    wait_for_journal_replay_complete
      get_master_journal_position
```
and in the end `test1/` gets passed as an image name to `rbd journal status` command:
```
2024-09-12T16:30:39.235 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:+ wait_for_replay_complete cluster1:0 cluster2 mirror test1
...
2024-09-12T16:30:39.588 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ get_master_journal_position cluster2 test1 ''
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local cluster=cluster2
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local pool=test1
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local image=
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ get_journal_position cluster2 test1 '' ''
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local cluster=cluster2
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local pool=test1
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local image=
2024-09-12T16:30:39.589 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local id_regexp=
2024-09-12T16:30:39.590 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:+++ mkfname cluster2-test1-.status
2024-09-12T16:30:39.590 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:+++ echo cluster2-test1-.status
2024-09-12T16:30:39.590 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:+++ sed -e 's|[/ ]|_|g'
2024-09-12T16:30:39.592 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ local status_log=/tmp/tmp.DobGEhjzn2/cluster2-test1-.status
2024-09-12T16:30:39.592 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ rbd --cluster cluster2 journal status --image test1/
2024-09-12T16:30:39.593 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:++ tee /tmp/tmp.DobGEhjzn2/cluster2-test1-.status
2024-09-12T16:30:39.638 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:rbd: error opening image test1/: (2) No such file or directory
2024-09-12T16:30:39.638 INFO:tasks.workunit.cluster1.client.mirror.smithi172.stderr:rbd: failed to open image test1/ to get journal name: (2) No such file or directory
```
https://qa-proxy.ceph.com/teuthology/dis-2024-09-12_15:21:08-rbd-wip-dis-testing-distro-default-smithi/7902669/teuthology.log

It looks like passing an additional `${POOL}` argument for `wait_for_replay_complete()` call in `rbd_mirror_ha.sh` was missed.